.travis.yml

```diff
@@ -52,8 +52,9 @@ script:
   - |
       if [[ $TESTS == 'blocks-examples' ]]; then
         cd $TRAVIS_BUILD_DIR
-        git clone https://github.com/mila-udem/blocks-examples.git
+        git clone https://github.com/laurent-dinh/blocks-examples.git
```

Comment:
In cases when a change in Blocks requires a change in Blocks-examples one typically create two PRs, with the Blocks one referring to the Blocks-examples one just like you do it here. But I don't see a companion PR in Blocks-examples.

@dwf , I thought the right way  to do cross-repository PR is explained [herehttp://blocks.readthedocs.org/en/latest/development/pull_request.html) but apparently it is not. Should we add such an explanation there? Or is it also present at some other page?